modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -218,17 +266,24 @@ struct Params
     }
 };
 
-// Base class for test fixtures
-template<typename ...SpecificParams>
-struct TestWithParamBase : TestFunctional,
-    TestWithParam<typename Params<SpecificParams...>::params_t>
+template<typename... SpecificParams>
+struct Params : public ParamsBase<std::tuple<MatType2,cv::Size,MatType2,cv::GCompileArgs(*)()>,
+                                  std::tuple<SpecificParams...>>
 {
-    using AllParams = Params<SpecificParams...>;
+    static constexpr const size_t compile_args_num = 3;
+};
 
-    MatType2 type = getCommonParam<0>();
-    cv::Size sz = getCommonParam<1>();
-    MatType2 dtype = getCommonParam<2>();
+template<typename ...SpecificParams>
+struct ParamsSpecific : public ParamsBase<std::tuple<cv::GCompileArgs(*)()>,
+                                               std::tuple<SpecificParams...>>
```

Comment:
Alignment. It may be better to put, as in [271](https://github.com/opencv/opencv/blob/8c0ae785562b7b52d1feba4fa2ed4e7f4d39b1ef/modules/gapi/test/common/gapi_tests_common.hpp) string.

Reply:
agree, fixed